app/code/Magento/Store/Console/Command/StoreListCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Magento\Store\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+
+/**
+ * Class StoreListCommand
+ * @package Magento\Store\Console\Command
```

Comment:
Please add more details in the description of the classes. Also, `@package` is not necessary.